element_event/event.py

```diff
@@ -93,17 +93,35 @@ class EventType(dj.Lookup):
     """Set of unique events present within a recording session
 
     Attributes:
-        event_type ( varchar(16) ): Unique event type.
+        event_type ( varchar(32) ): Unique event type.
         event_type_description ( varchar(256) ): Event type description.
     """
 
     definition = """
-    event_type                : varchar(16)
+    event_type                : varchar(32)   # short string identifier for a singular event type
```

Comment:
does `event_type` get propagated far down through the pipeline? If so, it would be great to keep these ids shorter. But if it's only used as a secondary attribute, then it's fine.

Reply:
It's only used as secondary attribute - and the longer varchar is a specific requirement for the CIM project